apps/site/src/content.config.ts

```
@@ -0,0 +1,10 @@
+import { defineCollection } from 'astro:content';
+import { docsLoader } from '@astrojs/starlight/loaders';
+import { docsSchema } from '@astrojs/starlight/schema';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)